stop-chromedriver.sh

```
@@ -0,0 +1,56 @@
+echo "=============================================="
+
+# Check if ChromeDriver is running
+CHROMEDRIVER_PIDS=$(pgrep -f chromedriver)
+
+if [ -z "$CHROMEDRIVER_PIDS" ]; then
+    echo -e "${YELLOW}ℹ️  No ChromeDriver processes found running${NC}"
+    exit 0
+fi
+
+echo -e "${GREEN}📋 Found ChromeDriver processes:${NC}"
+ps aux | grep chromedriver | grep -v grep
```

Shellcheck (0.10.0)

[info] Consider using pgrep instead of grepping ps output.

(SC2009)